Standards/CCSMB-X.md

```diff
@@ -0,0 +1,30 @@
+# CCSMB-X: Virtual File System Archive
+
+*Author: walksanator (Walker Fowlkes)*
+
+*Version: 1.0.*
+     
+*Last revised: 2023-01-DD*
+
+## Quick information
+
+| Information |                              |
+| ----------- | -----------------------------|
+| Version     | 1.0.9                        |
+| Author      | walksanator (Walker Fowlkes) |
+| Type        | Virtual File System Archiv   |
+| MIME        | `appilcation/vfs`            |
+| Extensions  | `.vfs`                       |
+
+## Technical details
+
+The VFS file format is a text-based format that represents folder(s) as a lua table
+where the keys of the table are used as file names and values are used for conents.
+Directories in VFS are tables, whereas files have their contents embedded as a string.
+VFS files should be read and written in binary mode as to prevent converting binary charachters to `?`
+One drawback of the VFS format is that it does not save any MetaData of files compressed
+
+## Examples included
```

Comment:
I'd recommend having the examples embedded inline with descriptions of what they do. This makes it easier to read quickly.

Reply:
but it is a full program, or should i have the Examples section contains example functions for creating/extracting (the gen_disk and unpack_vfs functions) along wit a small text example
and have the full example program and example image in the folder

Reply:
You shouldn't be posting the full program here - put that on your own repo and link to it as an additional example. The examples here should be limited to single functions that demonstrate just one part at a time.

Reply:
so should i embed the example functions i use to create/unpack the archive with comments
but *just* those functions
and include a simpler example since having a folder and 1 lua script and a 18kb text file is a bit mutch

Reply:
Sure, but try to make them as simple and easy to read as possible (e.g. assume the file's correct, no error checking).

Reply:
thats what they do allready
i do all the error checking outside of the functions anyway

Reply:
i think it is better now